analyzers/util.go

```diff
@@ -62,6 +62,9 @@ func newIssue(analyzerID string, desc string, fileSet *token.FileSet,
 	pos token.Pos, severity, confidence issue.Score,
 ) *issue.Issue {
 	file := fileSet.File(pos)
+	if file == nil {
+		return &issue.Issue{}
+	}
```

Comment:
🎨 A comment would be appreciated, because it doesn't sound logic to have this.

It does when reading issue, but once merged it won't I think

Reply:
Done, thanks